judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/submission/bundle/ContestantAnswerSummaryResponse.java

```diff
@@ -0,0 +1,22 @@
+package judgels.uriel.api.contest.submission.bundle;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.List;
+import java.util.Map;
+import judgels.jophiel.api.profile.BasicProfile;
+import judgels.sandalphon.api.submission.bundle.ItemSubmission;
+import judgels.uriel.api.contest.submission.ContestSubmissionConfig;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableContestantAnswerSummaryResponse.class)
+public interface ContestantAnswerSummaryResponse {
+    BasicProfile getProfile();
```

Comment:
Why basic? This should be `Profile`, like everything else, so that user rating is also returned.

Reply:
BasicProfile contains username, name, rating, and country already

Reply:
... just realized Profile contains username, rating and country. Will be changed to Profile.